src/sidebar/store/modules/selection.js

```diff
@@ -204,14 +203,18 @@ const update = {
     return setTab(state.selectedTab, action.tab);
   },
 
+  /**
+   * Automatically select the Page Notes tab, for convenience, if all of the
+   * top-level annotations in `action.annotations` are Page Notes and the
+   * previous annotation count was 0 (i.e. collection empty).
+   */
   ADD_ANNOTATIONS(state, action) {
-    const noteCount = arrayUtil.countIf(
-      action.annotations,
-      metadata.isPageNote
+    const topLevelAnnotations = action.annotations.filter(
+      annotation => !metadata.isReply(annotation)
     );
-    // If there are no annotations at all, ADD_ANNOTATIONS will not be called.
-    const haveOnlyPageNotes = noteCount === action.annotations.length;
-    // If this is the init phase and there are only page notes, select the page notes tab.
+    const noteCount = countIf(action.annotations, metadata.isPageNote);
+
+    const haveOnlyPageNotes = noteCount === topLevelAnnotations.length;
     if (action.currentAnnotationCount === 0 && haveOnlyPageNotes) {
```

Comment:
The `currentAnnotationCount` annotation-state property makes me raise an eyebrow.

Reply:
AFAIK this exists to work around the issue that each reducer can only see the state from the same module. Having this action property is not ideal though as we're essentially exposing an internal implementation detail of the store in an action payload, and there is the possibility of an error leading to this value being incorrect.

I think what this is telling us is that this particular reducer should be hoisted up a level (ie. not part of this module) as it depends on the state from multiple modules.

Anyhow, that's definitely an exercise for a separate PR!